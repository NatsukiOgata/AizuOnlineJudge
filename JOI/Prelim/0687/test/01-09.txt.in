100
24 65 75 84 94 172 211 227 229 247 259 315 374 383 412 449 564 566 571 611 619 638 639 661 787 788 804 806 811 843 936 1005 1032 1049 1050 1082 1142 1156 1209 1245 1327 1358 1361 1464 1473 1498 1756 1869 1949 1953 1934 1922 1920 1912 1871 1820 1812 1744 1741 1734 1620 1530 1482 1400 1384 1356 1353 1333 1229 1224 1215 1182 1159 1125 1118 1045 1043 1041 1009 978 953 897 895 889 805 762 662 594 588 585 558 504 455 323 314 120 108 52 46 45
