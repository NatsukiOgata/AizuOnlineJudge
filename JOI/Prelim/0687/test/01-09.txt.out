38289
53391
